SwiftExample/Podfile

```diff
@@ -4,9 +4,6 @@
  use_frameworks!
 
 target 'SwiftExample' do
-
-pod 'JSQMessagesViewController'
-
-
+	pod 'JSQMessagesViewController', :path => '../'
```

Comment:
No, this was actually intended: this ensures that the Swift demo project builds against the current version of the library. Otherwise (in particular because the Swift demo project is not CI-tested) you could commit changes that break the Swift demo app's build.